services/ui-src/src/components/menus/Menu.tsx

```diff
@@ -27,6 +27,7 @@ export const Menu = ({ handleLogout }: Props) => {
           rightIcon={<Icon icon="chevronDown" color="palette.white" />}
           sx={sx.menuButton}
           className={mqClasses}
+          aria-label="manage my account"
```

Comment:
Added `aria-label` to fix accessibility issue at mobile size ("My account" text is only shown on desktop).